filebeat/include/fields.go

```diff
@@ -32,5 +32,5 @@ func init() {
 // AssetFieldsYml returns asset data.
 // This is the base64 encoded zlib format compressed contents of fields.yml.
 func AssetFieldsYml() string {
```

Comment:
Hey. Why *.go files are touch? I see some more files are modified and in some unexpected parts.